spec/amp-email-format.md

```diff
@@ -0,0 +1,379 @@
+<!---
+Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# AMPHTML Email
+
+AMP is a technology commonly known for developing super fast web pages on mobile clients. AMP is in fact a set of HTML tags backed by JavaScript that allow all kinds of functionality with a focus on performance and security.
+
+There are [AMP components](https://www.ampproject.org/docs/reference/components) for everything from carousels, to responsive form elements, to retrieving fresh content from remote endpoints. The AMPHTML Email format provides a subset of AMP components that you can use in email messages. Recipients of AMP emails can view and interact with the AMP components directly in the email message.
+
+## FAQ
+
+* What is the official name of this technology?
+
+    * "AMPHTML Email" is the name of this technology.
+
+* Will this allow people to send JavaScript email?
+
+    * No. Valid AMP documents are not permitted to have their own JavaScript.
+
+* I heard that [AMP is going to support JavaScript](https://github.com/ampproject/amphtml/issues/13471). Will this be incorporated into AMPHTML Email?
+
+    *  No. To maintain users’ expectations of security and privacy, we’ll only allow a conservative subset of AMP functionality.
+
+* Are all AMP components supported?
+
+    * No. AMPHTML Email supports a restricted subset of the AMP components in order to ensure user safety in web-based mail clients.
+
+* Which AMP components will be supported?
+
+    * See the [AMP Components](#amp-components) section in the spec below.
+
+* Will the `amp-ad` component be supported?
+
+    * No.
+
+* Will the `amp-analytics` component by supported?
+
+    * No.
+
+* Do AMPHTML Email documents use AMP Caches?
+
+    * No.
+
+* Will this break email as an archival format?
+
+    * No, the spec still requires senders to include a static, HTML version of the email that users can view if they choose.
```

Comment:
Does the spec actually require/ mandate the presence of other versions?

Reply:
Maybe it should, we keep mentioning the HTML version as fall back in many cases and many email clients will need it as well.

Reply:
Good question. Maybe there are two views of what "spec" means:

1. The `amp4email` validator format
2. The AMPHTML email format (includes the new MIME part definition etc.) 

(1) can't actually enforce this but we can define this as "required" part of (2). Maybe it's worth spelling this out more clearly. /cc @adewale

Reply:
The validator format describes the set of properties that compliant AMPHTML email _must_ support in order to be valid whilst the written spec also describes the set of properties that compliant AMPHTML email _should_ support.

We should add an explicit line requiring that compliant AMPHTML email _should_ support other formats and saying that the reason this isn't a 'must' requirement is that there's no way to validate that the other MIME types have been added and include meaningful information.